OpenTabletDriver.Configurations/Configurations/Huion/H610.json

```
@@ -0,0 +1,70 @@
+{
+    "Name": "Huion H610",
+    "Specifications": {
+      "Digitizer": {
+        "Width": 203.2,
+        "Height": 127.0,
+        "MaxX": 32000.0,
+        "MaxY": 20000.0
+      },
+      "Pen": {
+        "MaxPressure": 2047,
+        "ButtonCount" : 2
+      },
+      "AuxiliaryButtons": null,
+      "Touch": null
+      },
+      "DigitizerIdentifiers" : [
+      {
+        "VendorID": 9580,
+        "ProductID": 110,
+        "InputReportLength": 2,
+        "OutputReportLength": null,
+        "ReportParser": "OpenTabletDriver.Configurations.Parsers.UCLogic.UCLogicReportParser",
+        "FeatureInitReport": null,
+        "OutputInitReport": null,
+        "DeviceStrings": {  
+      "6": "Huion",
+          "121": "M508"
+        }
+      },
+      {
+        "VendorID": 9580,
+        "ProductID": 110,
+        "InputReportLength": 3,
+        "OutputReportLength": null,
+        "ReportParser": "OpenTabletDriver.Configurations.Parsers.UCLogic.UCLogicReportParser",
+        "FeatureInitReport": null,
+        "OutputInitReport": null,
+        "DeviceStrings": {  
+      "6": "Huion",
+          "121": "M508"
+        },
+        "InitializationStrings": [
+          100,
+          123
+        ]
+      },
+       {
+        "VendorID": 9580,
+        "ProductID": 110,
+        "InputReportLength": 8,
+        "OutputReportLength": null,
+        "ReportParser": "OpenTabletDriver.Configurations.Parsers.UCLogic.UCLogicReportParser",
+        "FeatureInitReport": null,
+        "OutputInitReport": null,
+        "DeviceStrings": {
+      "6": "Huion",
+      "121" : "M508"
+        },
+        "InitializationStrings": [
+          100,
+          123
+        ]
+      }
+  ],
+    "AuxilaryDeviceIdentifiers": [],
```

The configuration spec is different on master than it is on `0.6.x`.
```suggestion
    "AuxiliaryDeviceIdentifiers": [],
```